test/uk/gov/hmrc/mobilepayments/connectors/OpenBankingConnectorSpec.scala

```diff
@@ -30,24 +30,23 @@ import java.time.LocalDateTime
 import scala.concurrent.Future
 
 class OpenBankingConnectorSpec extends BaseSpec with ConnectorStub with MobilePaymentsTestData with ScalaFutures {
-  val mockHttp:                   HttpClient    = mock[HttpClient]
-  implicit val mockHeaderCarrier: HeaderCarrier = mock[HeaderCarrier]
 
-  val sut           = new OpenBankingConnector(mockHttp, "baseUrl")
+
+  val sut           = new OpenBankingConnector(mockHttp, "https://baseUrl")
   val sessionDataId = "51cc67d6-21da-11ec-9621-0242ac130002"
   val returnUrl     = "https://tax.hmrc.gov.uk/payment-result"
   val amount        = BigDecimal((12500 * 100).longValue())
 
   "when getBanks call is successful it" should {
     "return banks" in {
-      performSuccessfulGET(Future successful banksResponse)(mockHttp)
+      performGET(Future successful banksResponse)(mockHttp, mockRequestBuilder)
```

Comment:
You can remove (mockHttp, mockRequestBuilder) once they are made as implicit  in BaseSpec